storm-server/src/main/java/org/apache/storm/scheduler/resource/strategies/scheduling/BaseResourceAwareStrategy.java

```diff
@@ -527,13 +538,13 @@ protected SchedulingResult scheduleExecutorsOnNodes(List<ExecutorDetails> ordere
             if (execIndex == 0) {
                 break;
             } else {
-                searcherState.backtrack(execToComp, nodeForExec[execIndex - 1], workerSlotForExec[execIndex - 1]);
```

Comment:
why did this line change?

Reply:
This method signature was changed to backtrack correctly in the presence of Acker executors. I believe this is Ray's change that did not make it to community earlier.

Reply:
Are there more missing changes that go along with this?  Can we get that merged first before this PR if there are more changes?